src/containerapp/azext_containerapp/tests/latest/test_containerapp_commands.py

```diff
@@ -1278,12 +1278,6 @@ def test_containerapp_addon_binding_e2e(self, resource_group):
             JMESPathCheck('length(@)', 0),
         ])
 
-        self.cmd('containerapp list -g {} --environment {}'.format(env_rg, env_name), checks=[
-            JMESPathCheck('length(@)', 0),
-        ])
-
-        self.cmd(f'containerapp env delete -g {env_rg} -n {env_name} --yes')
```

Comment:
May I ask why these two commands are removed from the test?

Reply:
Because the env is from `env_id = prepare_containerapp_env_for_app_e2e_tests(self, location=location)`, which will be reused by other test cases. 
So we should not delete it in this test case and should not check how many containerapps in the environment. That made this test case not stable when running with other test cases lively.